dss/storage/checkout.py

```diff
@@ -131,3 +132,34 @@ def touch_test_file(dst_bucket: str, replica: Replica) -> bool:
         return True
     except Exception as e:
         return False
+
+def status_file_name(execution_id: str) -> str:
+    return f"checkout/status/{execution_id}.json"
+
+def put_status_succeeded(execution_id: str, dst_replica: Replica, dst_bucket: str, dst_location: str):
+    handle = Config.get_blobstore_handle(Replica.aws)
```

Comment:
IMO this is unnecessarily confusing.  I would just put it at the dst_replica rather than centralize on AWS.